pkg/apis/operator/v1/types.go

```diff
@@ -55,6 +55,10 @@ type InstallationSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Standalone,Management,Managed
 	ClusterManagementType ClusterManagementType `json:"clusterManagementType,omitempty"`
+
+	// NodeMetricsPort allows users to specify which port calico/node serves metrics on or "nil" to disable it.
```

Comment:
These comments appear in our docs as well. I'd suggest avoiding the word `nil` because it is go-specific whereas these comments apply to the yaml as well, and avoid referring to the 'user' as our other documentation does not.
```suggestion
	// NodeMetricsPort specifies which port calico/node serves metrics on. If omitted, then metrics are disabled.
```